src/pages/Registration.tsx

```diff
@@ -143,14 +141,10 @@ const Registration = () => {
 
     return (
         <FullPageDialog>
-            <Card
-                raised={false}
-                sx={{
-                    background: 'transparent',
-                }}
-            >
-                <CardContent
+            <>
+                <Box
                     sx={{
+                        mb: 5,
```

Comment:
Increasing the margin of the `Typography` wrapper to account for the spacing lost with the removal of `CardContent`.